CMakeLists.txt

```diff
@@ -1,6 +1,18 @@
 #---Check if cmake has the required version-----------------------------------------------------
+
 cmake_minimum_required(VERSION 3.4.3 FATAL_ERROR)
-cmake_policy(SET CMP0005 NEW)
+
+#---CMake policy settings-----------------------------------------------------------------------
+# See "cmake --help-policy <POLICY>" for more details
+
+cmake_policy(VERSION 3.4.3)
```

Comment:
Do we need this? `cmake_minimum_required` is already calling this.

Reply:
Good point. I removed the call to `cmake_policy()`.